### PR TITLE
implement GET+POST `cluster/{cluster}/requests` 

### PR DESCRIPTION
**Pull request by Bee-lee**
# Description
- implement GET `cluster/{cluster}/requests`
- implement POST `cluster/{cluster}/requests`
- add missing `operationId` in v2 OpenAPI spec for POST `cluster/{cluster}/requests`
- Redis function to get details of Requests uses Redis pipelines so that our client only makes 1 request to the server to avoid multiple round trips
- because of ^ point, Redis is now a direct dependency

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Unit tests (no changes in the code)
- REST API tests

## Testing steps
local testing with real Redis with data according to spec doc

## Checklist
* [x] `make before_commit` passes
* [x] updated documentation wherever necessary
* [x] added or modified tests if necessary
* [x] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

**Review by juandspy — APPROVED** [review]

File: server/router_utils.go [hunk 1]
```diff
@@ -213,3 +214,40 @@ func readRequestID(writer http.ResponseWriter, request *http.Request) (types.Req
 
 	return validatedRequestID, nil
 }
+
+func readRequestIDList(writer http.ResponseWriter, request *http.Request) (
+	[]types.RequestID, error,
+) {
+	var requestList []string
```
Comment (tisnik) [inline server/router_utils.go:221]: you can pre-allocate by `make`

**Review by tisnik — CHANGES_REQUESTED** [review]
LGTM in overall, just one comment.

File: server/router_utils.go (hunk 1 above)
Comment (Bee-lee) [reply server/router_utils.go:221]: done

**Review by tisnik — APPROVED** [review]
LGTM

**Pull request merged**